src/social-network-provider/facebook.com/ui-provider.ts

```diff
@@ -44,7 +44,7 @@ export const facebookUISelf = defineSocialNetworkUI({
         facebookUISelf.requestPermission().then(granted => {
             if (granted) {
                 setStorage('facebook.com', { forceDisplayWelcome: true })
-                location.href = 'https://facebook.com/'
+                location.assign('https://www.facebook.com/')
```

Comment:
Is there a speical reason to use assign? e.g. if not, there is a bug in E2E test?

Reply:
It's from @septs ' advice. I think it's easier for spying.